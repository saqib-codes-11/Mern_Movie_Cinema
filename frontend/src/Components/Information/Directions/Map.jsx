
import GoogleMaps from "simple-react-google-maps"
import config from "../../../config.js"

const Map = ({ coord }) => {
    const { lat, lng } = coord;
    return (
        <GoogleMaps
            apiKey={config.googleMapsAPIKey}
            style={{ height: "300px", width: "100%" }}
            zoom={17}
            center={{ lat: lat, lng: lng }}
            markers={{ lat: lat, lng: lng }}
        />
    );
}


export default Map