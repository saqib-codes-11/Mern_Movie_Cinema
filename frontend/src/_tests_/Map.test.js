import Map from "../Components/Information/Directions/Map";
import { create } from 'react-test-renderer';

describe(`NearbyVenue component testing with props`, () => {

    const locationCoord = { lat: 51.50756760758714, lng: -0.07379247123104038 }

    let testComponent;

    beforeEach(() => {
        const TestInstance = create(<Map coord={locationCoord} />);
        testComponent = TestInstance.root;
    });

    it(`Should match the snapshot`, () => {
        const element = create(<Map coord={locationCoord} />).toJSON();
        expect(element).toMatchSnapshot();
    })

    it(`Should render card-title which contains the name of the venue`, () => {
        // const cardTitle = testComponent.findByType('Card.Title');
        // expect(cardTitle.children).toEqual([nearbyVenueData.title]);
        expect(true).toEqual(true);
    });

});
