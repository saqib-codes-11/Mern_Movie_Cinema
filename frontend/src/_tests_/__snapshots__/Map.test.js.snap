// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NearbyVenue component testing with props Should match the snapshot 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "300px",
      "width": "100%",
    }
  }
/>
`;
